ui/js/models/perf/alerts.js

```diff
@@ -18,15 +18,12 @@ treeherder.factory('PhAlerts', [
         };
         Alert.prototype.getGraphsURL = function (timeRange, alertRepository,
                                                 performanceFrameworkId) {
-            var signature = this.series_signature.signature_hash;
-            var url = "#/graphs?timerange=" + timeRange +
-                "&series=[" + [alertRepository, signature, 1] + "]" +
-                "&selected=[" + [alertRepository, signature] + "]";
+            let url = `#/graphs?timerange=${timeRange}&series=${alertRepository},${this.series_signature.id},1`;
 
             // for talos only, automatically add related branches
             if (performanceFrameworkId === 1) {
                 const branches = (alertRepository === "mozilla-beta") ? ['mozilla-inbound'] : thPerformanceBranches.filter(branch => branch !== alertRepository);
-                url += branches.map(branch => `&series=[${branch},${signature},0]`).join("");
+                url += branches.map(branch => `&series=${branch},${this.series_signature.signature_hash},0`).join("");
```

Comment:
I'm assuming that "this.series_signature.signature_hash" is required here (and not signature id) since we already know the framework is talos and we are looking for related branches (?)

Reply:
Yes, we're using the fact that signature hashes are consistent across repositories to quickly query for the same data on a different branch. If we were ever to remove signature hashes entirely we would need another strategy here. I'll add a comment about this.